internal/flyerr/flyerr.go

```diff
@@ -90,6 +90,7 @@ func PrintCLIOutput(err error) {
 	suggestion := GetErrorSuggestion(err)
 
 	if description != "" {
+		fmt.Println()
```

Comment:
@billyb2 I added a new line here.

I don't have my GOPATHs configured correctly (or I just don't understand how Golang works anymore) so I couldn't test this locally because my code is importing `"github.com/superfly/flyctl/internal/flyerr"` from some other path (or from source), but 

I'm hoping:
```
> go run . secrets deploy -a secrets-deploy
Error: no machines available to deploy
'fly secrets deploy' will only work if the app has been deployed and there are machines available

Try 'fly deploy' first
```

will be:

```
> go run . secrets deploy -a secrets-deploy
Error: no machines available to deploy

'fly secrets deploy' will only work if the app has been deployed and there are machines available

Try 'fly deploy' first
```

Reply:
```
Error: no machines available to deploy
'fly secrets deploy' will only work if the app has been deployed and there are machines available

Try 'fly deploy' first
```

Very nice!